package.json

```diff
@@ -43,7 +45,8 @@
     "toastr"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.0.0"
+    "ember-cli-babel": "^5.0.0",
+    "toastr": "^2.1.2"
```

Comment:
Just for toastr, not for ember-cli-babel. That needs to stay for transpilation of the addon.